FAQ.md

```diff
@@ -61,7 +61,9 @@ Based on [#1002](https://github.com/iptv-org/iptv/issues/1002), [#7415](https://
 ### Why don't we show NSFW channels in most playlists?
 NSFW content are not suitable for the main public. To avoid issues with these kind of programmings, we've created a separate NSFW playlist available for everyone. If you want to see this type of channel, please use the NSFW playlist.
 
-Based on [#1416](https://github.com/iptv-org/iptv/pull/1416) (output of NSFW in general playlist), [#2126](https://github.com/iptv-org/iptv/issues/2126) (output of NSFW, again), [#2234](https://github.com/iptv-org/iptv/issues/2234) (request of hide NSFW channels), [#4768](https://github.com/iptv-org/iptv/pull/4768) (separate some channels from others) and [#7245](https://github.com/iptv-org/iptv/pull/7245) (other issue).
+If NSFW channels is displayed by mistake, we suggest that you report via issue or edit their entries from [Database](https://github.com/iptv-org/database) repository.
```

Comment:
```suggestion
If NSFW channels are displayed by mistake, we suggest that you report via an issue or edit their entries from the [Database](https://github.com/iptv-org/database) repository.
```

Reply:
Fixed. I added this sentence to explain some error in the playlists, but I forgot to add "in the general playlist" for more clear.